### ISSUE TITLE
Do not install files directly to site-packages

### ISSUE THREAD

**Issue by mtelka** [miketheman/pytest-socket#230]
Currently there are two files installed directly to site-packages:
```
LICENSE
README.md
```
Please do not do so.

This is similar to [this pytest-freezer issue](https://github.com/pytest-dev/pytest-freezer/pull/11).

**Comment by miketheman** [comment]
Can you explain why this is a negative behavior, or where these are being installed and via which method?

**Comment by mtelka** [comment]
Files are installed here:
```
$ pip install pytest-socket==0.5.1
...
$ ls -l ~/.local/lib/python3.9/site-packages/
total 17
drwxr-xr-x 2 marcel staff    3 Jun 21 11:52 __pycache__
-rw-r--r-- 1 marcel staff 1080 Jun 21 11:52 LICENSE
drwxr-xr-x 2 marcel staff    9 Jun 21 11:52 pytest_socket-0.5.1.dist-info
-rw-r--r-- 1 marcel staff 6036 Jun 21 11:52 pytest_socket.py
-rw-r--r-- 1 marcel staff 5248 Jun 21 11:52 README.md
$
```
My scenario is this:

I package Python projects for OpenIndiana and I was hit by this problem when I updated the `pytest-freezer` package from version 0.4.6 to 0.4.7.  The `pytest-freezer` version 0.4.7 was not installable together with `pytest-socket` because they both tried to deliver `LICENSE` file at the same location (the OpenIndiana package manager refuses to install such conflicting packages).  The `pytest-freezer` project already fixed the issue/regression by releasing 0.4.8, but it would be great if `pytest-socket` also stops to pollute the top level `site-packages` directory.  Currently it causes no immediate trouble for us so it does not need any urgent action.  It would be just great to have it solved for future releases.

For reference, here is the error message produced by the package manager:
```
pkg update: The following packages all deliver file actions to usr/lib/python3.9/vendor-packages/LICENSE:

  pkg://openindiana.org/library/python/pytest-freezer-39@0.4.7,5.11-2023.0.0.0:20230619T061600Z
  pkg://openindiana.org/library/python/pytest-socket-39@0.6.0,5.11-2023.0.0.0:20230221T175354Z

These packages cannot be installed together. Any non-conflicting subset
of the above packages can be installed.
```

Thank you.
